doc/_config.adoc

```diff
@@ -44,6 +44,10 @@ ifndef::include-only-once[]
 // :images_osi_sensor_model_packaging: ./osi-sensor-model-packaging/doc/images  // example
 :imagesoutdir: ./images/generated_images
 
+:source-highlighter: pygments
```

Comment:
This will not work in Antora. Antora only supports hightlightjs directly. Anything else has to be implemented manually.

See also: https://docs.antora.org/antora/latest/asciidoc/source/#source-highlighter